src/xtgeo/surface/_regsurf_boundary.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+import xtgeo
+
+
+def create_boundary(self, alpha_factor, is_convex, simplify):
+    """Create boundary polygons for a surface."""
+
+    # make into points
+    points = xtgeo.points_from_surface(self)
+
+    # compute minimum alpha based on surface resolution
+    alpha = np.ceil(np.sqrt(self.xinc**2 + self.yinc**2) / 2)
+
+    alpha_factor = 0 if is_convex else alpha_factor
+
+    alpha *= alpha_factor
```

Comment:
also this one would be nice to do inside the` xtgeo.Polygons.boundary_from_points` function 🙂 (if possible to estimate minimum alpha from an pointset)

Reply:
Done